### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sophia-guo**
Signed-off-by: Sophia Guo <sophiag@ca.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
 
 The test infrastructure in this repository allows us to lightly yoke a great variety of tests together to be applied to testing the AdoptOpenJDK binaries.  It is a thin wrapper around a varied set of tests, to allow us to easily run all types of tests via make targets and as stages in our Jenkins CI pipeline builds.
 
+Note: there are additional changes coming to the testing at AdoptOpenJDK (re: top level make targets and possibly even node labels in Jenkins), so we will be in flux for a little while. Stay in tune.
+
```
Comment (smlambert) [inline README.md]: Please stay "Stay in tune" to "Stay tuned".

File: README.md [hunk 2]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
-Different jvmversion: openjdk9-openj9 | openjdk9 | openjdk8
-Different javaversion: SE80 | SE90
-## 0. Configure environment and get dependencies:
+Platform: x64_linux | x64_mac | s390x_linux | ppc64le_linux | aarch64_linux
+
+Javaversion: SE80 | SE90
+
+Prerequisites:
+	
+    perl 5.10.1 or above with JSON and Text::CSV module installed
```
Comment (smlambert) [inline README.md]: Eventually we will also need the XML::Parser, but can update when that change comes in.

File: README.md [hunk 3]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+    make 3.81 or above (recommend make 4.1 or above on windows)
+    wget 1.14 or above
+    ant 1.7.1 or above with ant-contrib.jar
+
+## Jenkins setup and running
+AdoptOpenjdk set up test builds against adoptopenjdk sdk builds, release sdk, nightly sdk.
```
Comment (smlambert) [inline README.md]: While you can run all the tests manually via the make targets on the command line, you may also run the tests in Jenkins.  As part of the AdoptOpenJDK continuous integration (CI), AdoptOpenJDK runs test builds against the release and nightly SDK builds.

File: README.md [hunk 4]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+
+Easily setting up your own test builds using AdoptOpenjdk-test Jenkinsfile by:
```
Comment (smlambert) [inline README.md]: You can easily set up your own Jenkins-based test builds using the AdoptOpenJDK openjdk-tests Jenkinsfile by:

File: README.md [hunk 5]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+	
+	* Configure machines using playbooks at https://github.com/AdoptOpenJDK/openjdk-infrastructure/tree/master/ansible/playbooks
+	* Set up similar machine labels
+	* Tar sdk as *.tar.gz
+
+### Setting up test build against SDK with URL :
+
+	* Create Pipeline test build job using Pipeline script from SCM :https://github.com/AdoptOpenJDK/openjdk-tests.git
```
Comment (smlambert) [inline README.md]: Create **a** Pipeline test build

File: README.md [hunk 6]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+	* Build with Parameters
+		* Choose SDK_RESOURCE customized
+		* Setting CUSTOMIZEDURL to your customized SDK URL
```
Comment (smlambert) [inline README.md]: CUSTOMIZED_SDK_URL vs CUSTOMIZEDURL

File: README.md [hunk 7]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+		* Setting all other parameters as PLATFORM, JAVA_VERSION, etc.
```
Comment (smlambert) [inline README.md]: parameters such as PLATFORM

File: README.md [hunk 8]
```diff
@@ -16,16 +16,58 @@ Why the need for other testing?  The OpenJDK regression tests are a great start,
...
+	* Create Pipeline test build job using Pipeline script from SCM :https://github.com/AdoptOpenJDK/openjdk-tests.git
+	* Configure your sdk build to trigger your created test build
+		* If using execute shell use Parameterized Trigger Plugin to passing in parameters including your sdk job name and job number
+		* If using pipeline update pipeline using following code : 
+
+```
+build job: 'test_build_name', parameters: [string(name: 'PLATFORM', value: '...'), string(name: 'JAVA_VERSION', value: '...'), ... string(name: 'UPSTREAM_JOB_NAME', value: "your sdk job name"), string(name: 'UPSTREAM_JOB_NUMBER', value: "your sdk job number")]
+```
+
+## Local setup and running
```
Comment (smlambert) [inline README.md]: Local setup and running (via make targets on the command line)

**Comment by sophia-guo** [comment]
Thanks @smlambert updated.

**Pull request merged**